Safety/Safety-Systems/AuxTel-Safety-Gate-Procedures.rst

```diff
@@ -0,0 +1,174 @@
+    - If a file cannot include a title (surrounded by ampersands (#)), comment out the title from the template and include a comment explaining why this is implemented (in addition to applying the ``title`` directive).
+
+.. Include one Primary Author and list of Contributors (comma separated) between the asterisks (*):
+.. |author| replace:: *Franco Colleoni, Jacqueline Seron*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *none*
+
+.. This is the label that can be used as for cross referencing this procedure.
+.. Recommended format is "Directory Name"-"Title Name"  -- Spaces should be replaced by hyphens.
+.. _AuxTel-Safety-Gate-Procedures:
+.. Each section should includes a label for cross referencing to a given area.
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link an explicit title using the syntax :ref:`link text <label-name>`.
+.. An error will alert you of identical labels during the build process.
+
+###############################
+AuxTel Safety Gate Procedures
+###############################
+
+.. _AuxTel-Safety-Gate-Procedures-Overview:
+
+Overview
+========
+
+.. This section should provide a brief, top-level description of the document's purpose and utilization. 
+
+AuxTel safety gate is a safety mechanism that prevents telescope and dome movement 
+when individuals pass through the gate to the 2nd floor.  
+It is controlled by the AuxTel Interlock System (AuxTel IS).
+
+This document outlines the following:
+
+* **Gate operation**: opening and closing procedures, 
+
+* **Recover the safety gate** in an Error state and in E-STOP, 
+
+* Verify the **status for the telescope and dome drives**.
+
+For more information refer to the :ref:`AuxTel Safety Gate Introduction <AuxTel-Safety-Gate-Introduction>`
+
+
+
+.. _AuxTel-Safety-Gate-Procedures-Gate-operation:
+
+Gate operation
+========================
+
+Gate status:
+
+* **Fully open**: :guilabel:`Release` pressed, handle removed, physically open.
+
+* **Fully closed**: physically closed, handle inserted, and :guilabel:`Lock` pressed.
+
+.. warning::
+    If :guilabel:`Lock` is NOT pressed the AuxTel IS will NOT register the gate as closed, even when is physically closed,
+    and the telescope and dome drives will remain disabled. 
+
+    If :guilabel:`Release` is pressed the AuxTel IS will register the gate as open, even if the gate is physically closed.
+
+.. Fig3. Gate fully opened 
+.. Fig4. Gate in closed position
+
+
+.. figure:: ./_static/gate-fully-open-and-gate-fully-closed.png
+      :width: 500px
+      
+      Safet gate: fully opened (left), fully closed (right)
```

Comment:
The image of "Fully Closed" doesn't show all four LEDs ON.

Reply:
good catch! I edited it, we may want to get another photo eventually.